.github/ISSUE_TEMPLATE/docs.md

```diff
@@ -3,21 +3,19 @@ name: Documentation
 about: Suggest an improvement to our documentation at material.angularjs.org
 labels: "type: docs, needs triage"
 ---
-# AngularJS Material is in LTS mode
+<!--
 
-We are no longer accepting changes that are not critical bug fixes into this project.
-See [Long Term Support](https://material.angularjs.org/latest/#long-term-support) for more detail.
+---------------------------------------------------------------------
+This repo is for AngularJS Material, not Angular Material or the CDK.
```

Comment:
nit: in official docs we try to say "Angular CDK" vs. "the CDK" (though we tend to refer to it that way in conversation)